### DISCUSSION TITLE
Transformations API using client credentials

### DISCUSSION THREAD

**Discussion by hretief** [iTwin/itwinjs-core#2626] [Q&A] [answered]
Been trying out the Transformations API.  Is the transformations API enabled for client credentials?  I created a client credential in the usual way on https://developer.bentley.com/my-apps/; choosing a "Service" type.

But when I execute the filterbyviewdefinition API "https://api.bentley.com/transformations/configurations/filterbyviewdefinition/ "  
I immediately get an error stating the the identity provider is wrong:

 | {"error":{"code":"Unauthorized","message":"IDX10205: Issuer validation failed. Issuer: 'https://imsoidc.bentley.com'. Did not match:
     | validationParameters.ValidIssuer: '' or validationParameters.ValidIssuers: 'https://ims.bentley.com'."}}

**Comment by aruniverse** [comment, marked as answer]
Take a look at the last bullet in https://github.com/iTwin/itwinjs-core/discussions/2158
